api/models/resource.go

```diff
@@ -27,6 +27,44 @@ type MembersResourceRelation struct {
 	Emails []string `json:"emails"`
 }
 
+// MemberAccess represents that a member has access to a certain resource.
+//  this will get pushed to a device.
+type MemberAccess struct {
+	Email           string
+	ResourceAddress string
+	ResourceName    string
+	Name            string
+	RFID            string
+}
+
+// ACLUpdateRequest is the json object we send to a resource when pushing an update
+type ACLUpdateRequest struct {
+	ACL []string `json:"acl"`
+}
+
+// ACLResponse Response from a resource that is a hash of the ACL that the
+//   resource has stored
+type ACLResponse struct {
+	Hash string `json:"acl"`
+	// Name of the resource - this should match what we have in the database
+	//  so we know which acl to compare it with
+	Name string `json:"name"`
+}
+
+type AddMemberRequest struct {
+	ResourceAddress string `json:"doorip"`
+	Command         string `json:"cmd"`
+	UserName        string `json:"user"`
+	RFID            string `json:"uid"`
```

Comment:
Maybe `json:"rfid"`

Reply:
it will break things if i change it
because of the rfid device